components/scream/scripts/gather_all_data.py

```diff
@@ -27,7 +27,7 @@ def formulate_command(self, machine):
         compiler  = get_mach_cxx_compiler(machine)
         batch     = get_mach_batch_command(machine)
 
-        env_setup_str = " && ".join(env_setup) + " CTEST_PARALLEL_LEVEL={}".format(get_mach_testing_resources(machine))
+        env_setup_str = " && ".join(env_setup) + " && CTEST_PARALLEL_LEVEL={}".format(get_mach_testing_resources(machine))
```

Comment:
Coming to think of it, when running on local machines, it *might* happen that env_setup is empty. In that case, you'd end with `env_setup_str = && CTEST_PARALLEL_LEVEL=...`, which is not valid. Mayve this is better:

```
env_setup.append("CTEST_PARALLEL_LEVEL={}".format(get_mach_testing_resources(machine)))
env_setup_str = " && ".join(env_setup)
```

Reply:
@bartgol , I made the change.  We should be good now.